v2.0/create-index.md

```diff
@@ -6,6 +6,8 @@ toc: false
 
 The `CREATE INDEX` [statement](sql-statements.html) creates an index for a table. [Indexes](indexes.html) improve your database's performance by helping SQL locate data without having to look through every row of a table.
 
+<span class="version-tag">New in v2.0:</span> To create an index for a column containing schemaless data (i.e., [`JSONB`](jsonb.html)), use an [inverted index](inverted-indexes.html).
```

Comment:
I know we'll eventually support other options for schemaless data, but since `JSONB` is the only option right now, this language feels ambiguous. Should we just say something like the following?

> To create an index on the schemaless data in a `JSONB` column, use...

Reply:
done